lib/hyphenated_words.dict.txt

```diff
@@ -0,0 +1 @@
+noun purpleair purple-air
```

Comment:
Do you think we should just strip /all/ hyphens and provide them as related terms? We could do that with Regex! (https://rubular.org) and `'string'.gsub(/.../)`

We might rename this and use it for more complex additional terms like `h2s: hydrogen-sulfide` as in the original issue.

Reply:
@jywarren sorry But I am not getting what you are trying to say. Also the link you provided is not loading 

:sweat_smile:
--

Reply:
Oops, http://rubular.com/ ! I meant we could do `'string'.gsub('-', '')` to remove hyphens pretty easily. I clarified in #4823 in a longer comment about the 3 cases, i hope that helps!

Reply:
Oops, http://rubular.com/ ! I meant we could do `'string'.gsub('-', '')` to remove hyphens pretty easily. I clarified in #4823 in a longer comment about the 3 cases, i hope that helps!

Reply:
Oops, http://rubular.com/ ! I meant we could do `'string'.gsub('-', '')` to remove hyphens pretty easily. I clarified in #4823 in a longer comment about the 3 cases, i hope that helps!

Reply:
OK, but so can we do one more thing and provide a second dict that's for non-hyphen-related transforms like in my comment:

`h2s: hydrogen-sulfide`

Otherwise this PR is good to merge!

Reply:
I mean that we can either have one file specifically for "adding hyphens in" and another for "related terms" (even though both work similarly) OR we should simply consider "adding hyphens" to be a subset of "related terms" and have a single file. I'm fine with either, just don't want to have to ask people to put hyphen-unrelated word pairs into a file that's labelled with relation to hyphens, make sense? 

Great!!!!